librz/core/casm.c

```diff
@@ -198,9 +203,17 @@ RZ_API RzCmdStatus rz_core_asm_plugins_print(RZ_NONNULL RZ_BORROW RzCore *core,
 				free(c);
 				break;
 			}
+			else if (!strcmp(arch, ap->name) && state->mode == RZ_OUTPUT_MODE_STANDARD) {
+				snprintf(header, sizeof(header), "NO CPUs for Arch[%s]", ap->name);
+				rz_cons_println(header);
+			}
 		}
 	} else {
 		rz_cmd_state_output_array_start(state);
+		if(state->mode == RZ_OUTPUT_MODE_STANDARD){
+			snprintf(header, sizeof(header), "%-8s %-10s %-11s %-10s %s", "Plugins", "Bits", "Arch Name", "License", "Description");
+			rz_cons_println(header);
+		}
```

Comment:
`RZ_OUTPUT_MODE_STANDARD` cannot have table headers. for that there is table mode that already implements it.

Reply:
Ah, didn't see a TABLE_MODE in rz-asm. Thank you for the heads up.